internal/webhook/admission_test.go

```diff
@@ -220,7 +220,7 @@ func TestCreateSpinAppWithMultipleExecutors(t *testing.T) {
 			Namespace: "default",
 		},
 		Spec: spinv1alpha1.SpinAppSpec{
-			Image:    "ghcr.io/spinkube/containerd-shim-spin/examples/spin-rust-hello:v0.13.0",
+			Image:    "ghcr.io/spinframework/containerd-shim-spin/examples/spin-rust-hello:latest",
```

Comment:
Do we still publish a versioned tag? - we don't use `:latest` usually for stability reasons